drivers/usbdev/cdcacm.c

```diff
@@ -265,12 +265,18 @@ static const struct uart_ops_s g_uartops =
 #ifdef CONFIG_SERIAL_IFLOWCONTROL
   cdcuart_rxflowcontrol, /* rxflowcontrol */
 #endif
+#ifdef CONFIG_SERIAL_TXDMA
```

Comment:
don't need since CONFIG_CDCACM select SERIAL_TX_DMA and SERIAL_RX_DMA in Kconfig

Reply:
I don't think this is wrong, it is a belt and suspenders type of solution (if the logic behind CONFIG_CDCACM changes, these don't need to be changed)